logging/net_appender.go

```diff
@@ -34,22 +35,20 @@ type CloudConfig struct {
 // NewNetAppender creates a NetAppender to send log events to the app backend. NetAppenders ought to
 // be `Close`d prior to shutdown to flush remaining logs.
 // Pass `nil` for `conn` if you want this to create its own connection.
-func NewNetAppender(config *CloudConfig, conn rpc.ClientConn) (*NetAppender, error) {
+// Pass `shutdownIters`=-1 for the default value.
```

Comment:
rm since we aren't adding `shutdownIters` anymore

Reply:
done in 2b4da5b